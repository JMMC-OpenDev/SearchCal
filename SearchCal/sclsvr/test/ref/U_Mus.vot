version https://git-lfs.github.com/spec/v1
oid sha256:1e2f22caeccb3d1f236f4f5f9911556a349e14ad17cf94112fe04d931fa7781f
size 153442
