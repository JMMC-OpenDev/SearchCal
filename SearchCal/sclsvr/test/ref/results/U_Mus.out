version https://git-lfs.github.com/spec/v1
oid sha256:3bc462d6348b9819527b24f751bbbe20ce4cabbeb77f8c98d940b3d629083fe5
size 201477
